src/vectorclock.erl

```diff
@@ -68,6 +114,51 @@ max([]) -> new();
 max([V]) -> V;
 max([V1,V2|T]) -> max([merge(fun erlang:max/2, V1, V2)|T]).
 
+-spec max_vc(vectorclock(), vectorclock()) -> vectorclock().
```

Comment:
Thank you for these comments. 
I thought about what you say here it made me realise I actually need the already present max and min behaviour.
I removed the max_vc and min_vc functions.